.github/workflows/package.yml

```diff
@@ -0,0 +1,293 @@
+name: Package and release
+
+on:
+  push:
+    tags:
+      - '*'
+
+env:
+  BUILD_DIR: Dist
+
+jobs:
+  linux:
+    runs-on: ubuntu-20.04
+    container: seladb/${{ matrix.image }}
+    strategy:
+      matrix:
+        include:
+          - image: ubuntu2204
+            config-zstd: OFF
+          - image: ubuntu2204-icpx
+            config-zstd: OFF
+            additional-flags: -DCMAKE_C_COMPILER=icx -DCMAKE_CXX_COMPILER=icpx
+          - image: ubuntu2004
+            config-zstd: OFF
+          - image: ubuntu1804
+            config-zstd: OFF
+          - image: centos7
+            config-zstd: OFF
+          - image: fedora37
+            config-zstd: OFF
+          - image: alpine317
+            config-zstd: OFF
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup Intel Compiler variables
+        if: contains(matrix.image, 'icpx')
+        run: |
+          . /opt/intel/oneapi/setvars.sh
+          printenv >> $GITHUB_ENV
+
+      - name: Debug Cmake
+        run: cmake --system-information
+
+      - name: Configure PcapPlusPlus
+        run: cmake -DPCAPPP_PACKAGE=ON -DLIGHT_PCAPNG_ZSTD=${{ matrix.config-zstd }} ${{ matrix.additional-flags }} -S . -B "$BUILD_DIR"
+
+      - name: Build PcapPlusPlus
+        run: cmake --build "$BUILD_DIR" -j
+
+      - name: Package
+        run: cmake --build "$BUILD_DIR" --target package
+
+      - name: Upload binaries to release
+        uses: ncipollo/release-action@v1
+        with:
+          allowUpdates: true
+          artifacts: ${{ env.BUILD_DIR }}/*.tar.gz
+
+  freebsd:
+    runs-on: macos-12
+    strategy:
+      matrix:
+        include:
+          - freebsd-version: "12.3"
+          - freebsd-version: "13.0"
+          - freebsd-version: "13.1"
```

Comment:
Do you think we need both 13.0 and 13.1? Shouldn't we choose just one of them?

Reply:
I don't know the release scheduled of FreeBSD, one artifact is ~9MB and this job is run only once per tag.

So up to you, we can keep only 13.1